<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>730</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(110, 122, 179, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>711</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.035, y1:0, x2:1, y2:0, stop:0.20398 rgba(172, 173, 179, 0), stop:0.975124 rgba(44, 83, 255, 0));
font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>App pensada para ayuda a contar el dinero en base a la cantidad de billetes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>621</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.035, y1:0, x2:1, y2:0, stop:0.20398 rgba(172, 173, 179, 0), stop:0.975124 rgba(44, 83, 255, 0));
font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>y Almacenar esos datos para su uso posterior</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.035, y1:0, x2:1, y2:0, stop:0.20398 rgba(172, 173, 179, 0), stop:0.975124 rgba(44, 83, 255, 0));
font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>Presiona &quot;Ayuda&quot; Para aprender a usar la aplicacion</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="recordFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>731</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="records_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>471</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 255);</string>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Moneda</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cantidad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cant. en Moneda</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="total_table">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 255);</string>
     </property>
     <row>
      <property name="text">
       <string>CUP</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="getRecords_Button">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 7pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>Update Record</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calculate_Button">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>15</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;
background-color: qlineargradient(spread:reflect, x1:0.035, y1:0, x2:1, y2:0, stop:0.20398 rgba(172, 173, 179, 0), stop:0.975124 rgba(44, 83, 255, 0));</string>
     </property>
     <property name="text">
      <string>History</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="help_Button">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>AYUDA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="otherApps_Button">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>320</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>Otras Apps</string>
     </property>
    </widget>
    <widget class="QPushButton" name="about_Button">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>320</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 10pt &quot;Open Sans Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>Acerca De</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
